packages/bpk-component-infinite-scroll/src/DataSource.js

```diff
@@ -21,13 +21,11 @@
 class DataSource<T = any> {
   listeners: Array<() => mixed>;
 
-  triggerListeners: (...args: Array<any>) => void;
```

Comment:
Out of interest why was this removed? Does it cause any issues?

Reply:
Yes! Was complaining that it's defined twice. I don't think we need it here as the method is defined and this was just for the type I assume?